flag_int_slice.go

```diff
@@ -136,7 +141,52 @@ func lookupIntSlice(name string, set *flag.FlagSet) []int {
 		if err != nil {
 			return nil
 		}
+		// extract default value from the flag
+		var defaultVal []int
+		for _, v := range strings.Split(f.DefValue, ",") {
+			iV, _ := strconv.Atoi(v)
+			defaultVal = append(defaultVal, iV)
+		}
+		// if the current value is not equal to the default value
+		// remove the default values from the flag
+		if !isIntSliceEqual(parsed, defaultVal) {
+			for _, v := range defaultVal {
+				parsed = removeFromIntSlice(parsed, v)
+			}
+		}
 		return parsed
 	}
 	return nil
 }
+
+func removeFromIntSlice(slice []int, val int) (newVal []int) {
```

Comment:
Looks like this one still has the old definition—can we update this to match the other remove functions?

Reply:
Done